sdk/src/calibration_96tof1.cpp

```diff
@@ -133,8 +265,16 @@ aditof::Status Calibration::getAfeFirmware(const std::string &mode,
     return Status::GENERIC_ERROR;
 }
 
-aditof::Status Calibration::getGainOffset(const std::string &mode, float &gain,
-                                          float &offset) const {
+//! getAfeFirmware - Get the depth gain ad offset values for a mode
+/*!
+getAfeFirmware - Get the depth gain ad offset values for a mode
```

Comment:
It should be getGainOffset, not getAfeFirmware

Reply:
Done